src/auth/auth.service.ts

```diff
@@ -40,6 +33,22 @@ export class AuthService {
     return userRecordWithoutPassword;
   }
 
+  private signUser(id: string, role: RoleType) {
```

Comment:
While you are here, you could give this function a better name, and update all references

```suggestion
  private signUserToken(id: string, role: RoleType) {
```

Reply:
Will do